features/facebookImport/src/popUps/infoPopUps/onOffFacebook.jsx

```diff
@@ -1,49 +1,50 @@
 import React from "react";
 
 import i18n from "../../i18n.js";
-import BaseInfoPopUp from "../baseInfoPopUp/baseInfoPopUp.jsx";
 import Infographic from "../../components/infographic/infographic.jsx";
 
 const OnOffFacebookInfoPopUp = () => {
-    const onOffFacebookInfoText = [
+    return (
```

Comment:
The original BaseInfoPop was expecting an array of children and had some logic on how to display them. The components in `infoPopUps` were also directly importing `BaseInfoPopUp` After the shuffling it wasn't necessary and it was getting in the way. What I've messed up was how I've structured the commit. I should have split it in two.

So, all the changes to every component in `infoPopUps` can be summarized as:
- remove BaseInfoPopUp
- remove the array and let the component just return the contents
- change the style around InfoGraphic so it's closer to the Figma design

I'm also considering renaming `infoPopUps` to `contents` I think it would make it a bit more clear, but I would like to hear some opinions before doing it.

Reply:
content is a bit generic, could be anything; if that's the case, maybe it's a good name. But I'm not good with naming, only with nitpicking name changes :-)